lib/profile/view_profile/sections/contact.dart

```diff
@@ -137,7 +137,7 @@ class ProfileContact extends StatelessWidget {
       MaterialPageRoute(
         builder: (context) => ChatMessenger(
```

Comment:
This widget is missing two required parameters - `groupChatId` and `id`.

Reply:
This is addressed in my "known bugs" listed above. It will be handled in a later PR